HospitalWebAppIntegrationTests/ExaminationCancelingTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using HospitalWebApp;
+using System.Net;
+using Xunit;
+
+namespace HospitalWebAppIntegrationTests
+{
+    public class ExaminationCancelingTests
+    : IClassFixture<HospitalWebApplicationFactory<Startup>>
+    {
+        private readonly HospitalWebApplicationFactory<Startup> _factory;
+        public ExaminationCancelingTests(HospitalWebApplicationFactory<Startup> factory)
+        {
+            _factory = factory;
+        }
+
+        [Fact]
+        public async void Cancel_examination_success()
+        {
+            var client = _factory.CreateClient();
+
+            var response = await client.GetAsync("/examination/cancel/2");
+
+            response.EnsureSuccessStatusCode();
```

Comment:
Is this regular assert call or?

Reply:
Yes, this throws an exception if status code of the response is not success (which is 2xx)